src/ontology/metadata/omim.yml

```diff
@@ -3,6 +3,24 @@ label: Online Mendelian Inheritance in Man
 prefix_map:
    OMIM: https://omim.org/entry/
    OMIMPS: https://omim.org/phenotypicSeries/PS
+   CHR: http://purl.obolibrary.org/obo/CHR_
+   CL: http://purl.obolibrary.org/obo/CL_
+   HGNC: "https://identifiers.org/hgnc:"
+   HGNC_symbol: https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/
+   IAO: http://purl.obolibrary.org/obo/IAO_
+   NCBIGene: https://www.ncbi.nlm.nih.gov/gene/
+   NCBITaxon: http://purl.obolibrary.org/obo/NCBITaxon_
+   ORPHA: http://www.orpha.net/ORDO/Orphanet_
+   PMID: http://www.ncbi.nlm.nih.gov/pubmed/
+   RO: http://purl.obolibrary.org/obo/RO_
+   SO: http://purl.obolibrary.org/obo/SO_
```

Comment:
@matentzn This is the relevant part for your review.

In order to expand SO, we need more than just `base_prefix_map` (the list of "owned" "native" ontology prefixes).

I considered two ways of doing this:
a. Just provide all of the prefixes that are inside the ontology itself, which is what I did here.
b. Try and just expand it without providing a prefix_map for it. https://github.com/biopragmatics/bioregistry#generating-iris

Maybe there's a different way you prefer. I went with (a) because it's safer. But I should note that now I've changed the exclusion term expansion script to use `prefix_map` instead of `base_prefix_map`. I just had to remove all references to `base_prefix_map` and change them to `prefix_map`. 

Questions
1. If you agree with (a), then we are no longer using `base_prefix_map`, so should we remove it?
2. Should I update `prefix_map` in all of the `metadata/ONTO.yml` to the full prefix_map in each of the ontologies?
3. If (2) is yes, then should we automate that? I wonder if OAK or some other tool has a way to extract the `prefix_map` from a given ontology file already.

Reply:
I had some previous thoughts in the above comment that I found incorrect and removed.

Reply:
a) was the correct way. The `prefix_map` should be used for expansions, the `base_prefix_map` is more a lookup which IRI prefixes belong to which ontology.. So you did it correctly.

> Should I update prefix_map in all of the metadata/ONTO.yml to the full prefix_map in each of the ontologies?

Not enough context for this - I would prefer a single prefix map for everything and base prefixes for each individual ontologies. Feel free to put on list and ask in meeting with some more context.

Reply:
That sounds like a good idea. Although, what if two different ontologies (let's just say ORDO and DO) are using 2 different versions / URIs for their prefixes for a given external ontology (let's just say RO for sake of example).

A way to fix this would be, for `prefix_map` to not be of type `Dict[str, str]`, but of type `Dict[str, List[str]` to accommodate for various different versions / URIs for each of these prefixes.

I'll bring up at the meeting.